swaglyrics/chrome.py

```diff
@@ -0,0 +1,14 @@
+# pyFile to keep Track of Global variables related to Chrome Extension.
+
+
+def initvariables():
+    global song, artist, isChrome, isTerminal
+    song = None
```

Comment:
since song, artist are initialised with `None`, on first run no lyrics would be displayed.

Reply:
Not necessarily. only in `isChrome` , it'll wait for response

Reply:
otherwise, works as usual

Reply:
I'm obviously talking about Chrome here

Reply:
need to add socket connection. will resolve it soon